tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc

```diff
@@ -699,6 +700,15 @@ TEST (nnstreamer_nnfw_mlapi, invoke_single_02_n)
   ml_tensors_info_destroy (in_res);
 }
 
+#define wait_for_sink(expected_cnt) \
+  do { \
+    while (sink_called_cnt < expected_cnt) { \
+      g_usleep (TEST_DEFAULT_SLEEP_TIME); \
+      waiting_time += TEST_DEFAULT_SLEEP_TIME; \
+      ASSERT_TRUE(TEST_TIMEOUT_LIMIT > waiting_time); \
```

Comment:
```ASSERT_TRUE(TEST_TIMEOUT_LIMIT > waiting_time);``` -->
```ASSERT_GT(TEST_TIMEOUT_LIMIT, waiting_time);``` provides better assertion messages.

Reply:
I agree with you. it's fixed